tests/callbacks_test.go

```diff
@@ -206,3 +206,25 @@ func TestPluginCallbacks(t *testing.T) {
 		t.Errorf("callbacks tests failed, got %v", msg)
 	}
 }
+
+func TestCallbacksRemoveAndGet(t *testing.T) {
+	db, _ := gorm.Open(nil, nil)
+	createCallback := db.Callback().Create()
+
+	err := createCallback.Register("plugin_1_fn1", c1)
+
```

Comment:
Hello, this blank line can be deleted?

Reply:
@demoManito Hello, thank you for the review. I have removed the blank line.